content/en/docs/concepts/containers/images.md

```diff
@@ -358,7 +365,8 @@ common use cases and suggested solutions.
    - Generate registry credential for each tenant, put into secret, and populate secret to each tenant namespace.
    - The tenant adds that secret to imagePullSecrets of each namespace.
 
-{{% /capture %}}
 
 If you need access to multiple registries, you can create one secret for each registry.
 Kubelet will merge any `imagePullSecrets` into a single virtual `.docker/config.json`
+
```

Comment:
@sftim , the changes are negligible, but seem fine. If you have time, you could address other style nits on the page. I realize that the page is likely to require further content changes at some point.
I read lines 369-370. These sentences seem out of place. Do they belong in the list or a different section of the page?
nit: Line 70, could start sentence with, The kubelet instead of Kubelet.